### PR TITLE
Add default conv variables

### PR DESCRIPTION
**Pull request by kevindougherty-noaa**
Added additional default variables that are included in conventional diags.

### CODE REVIEW

File: src/emcpy/cfg/var_defaults.yaml [hunk 1]
```diff
@@ -88,3 +88,33 @@ surface pressure:
...
+  short name: sst
+  units: K
+  variable type: conventional
+tc pressure:
+  cmap: rainbow
+  contour interval: 4
+  diff contour interval: 0.5
+  diff range: [-5, 5]
+  name: tc pressure
+  range: [800, 1050]
```
Comment (CoryMartin-NOAA) [inline src/emcpy/cfg/var_defaults.yaml]: in this YAML, surface pressure is in Pa, can we use Pa here too and just make sure to convert when plotting tcp?

File: src/emcpy/cfg/var_defaults.yaml [hunk 2]
```diff
@@ -88,3 +88,33 @@ surface pressure:
...
+bending angle:
+  cmap: viridis
+  contour interval: 0.001
+  diff contour interval: 0.01
+  diff range: [-2, 2]
+  name: bending angle
+  range: [0, 0.075]
+  short name: gps
+  units: rad
+  variable type: conventional
```
Comment (CoryMartin-NOAA) [inline src/emcpy/cfg/var_defaults.yaml]: says 'no newline at end of file', not sure why?

File: src/emcpy/cfg/var_defaults.yaml [hunk 3]
```diff
@@ -88,3 +88,33 @@ surface pressure:
...
+  name: tc pressure
+  range: [800, 1050]
+  short name: tcp
+  units: hPa
+  variable type: conventional
+bending angle:
+  cmap: viridis
+  contour interval: 0.001
+  diff contour interval: 0.01
+  diff range: [-2, 2]
```
Comment (CoryMartin-NOAA) [inline src/emcpy/cfg/var_defaults.yaml]: how is the diff range 2 orders of magnitude larger than the range?

**Review by CoryMartin-NOAA — COMMENTED** [review]
Few comments.

File: src/emcpy/cfg/var_defaults.yaml (hunk 2 above)
Comment (kevindougherty-noaa) [reply src/emcpy/cfg/var_defaults.yaml]: Interesting. I don't see that, but I will add the the extra line at the end which causes that.

File: src/emcpy/cfg/var_defaults.yaml [hunk 4]
```diff
@@ -100,21 +100,22 @@ sea surface temperature:
...
+  diff range: [-0.05, 0.05]
   name: bending angle
-  range: [0, 0.075]
+  range: [-0.001., 0.08]
   short name: gps
   units: rad
-  variable type: conventional
\ No newline at end of file
+  variable type: conventional
+
```
Comment (CoryMartin-NOAA) [inline src/emcpy/cfg/var_defaults.yaml:121]: lol it still says 'no new line at end of file' but now after the empty line... must be your text editor?

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**